nautobot/extras/tests/test_filters.py

```diff
@@ -1679,6 +1591,10 @@ def test_secret_type(self):
         params = {"secret_type": [SecretsGroupSecretTypeChoices.TYPE_PASSWORD]}
         self.assertEqual(self.filterset(params, self.queryset).qs.count(), 2)
 
+    @skip("Skipping SecretsGroupAssociation is a through model.")
+    def test_q_filter_exists(self):
+        """Test is not applicable for model."""
```

Comment:
Probably could add a q filter that checks `secrets_group__name`, `secret__name`, etc.?